### PR TITLE
Misc cleanup of all text files

### PR DESCRIPTION
**Pull request by basicfeatures**
- Remove CR+LF (^M)
- Remove trailing whitespace
- Compress two or more consecutive blank lines
- Ensure newline at EOF

https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

### CODE REVIEW

**Review by jarednorman — DISMISSED** [review]

File: dev_tools/lib/solidus/release_drafter.rb [hunk 1]
```diff
@@ -68,7 +68,6 @@ def save_release(draft, candidate_tag, branch)
     def full_changelog(current_diff_source_tag, candidate_tag)
       <<~TXT
 
-
```
Comment (kennyadsl) [inline dev_tools/lib/solidus/release_drafter.rb:71]: This extra space is intentional, see https://app.circleci.com/pipelines/github/solidusio/solidus/3952/workflows/a82c4d50-c4ee-426b-92b4-5cccee0ad4fe/jobs/36006. We can probably get rid of it, but the specs should reflect the change, cc @waiting-for-dev.

Comment (jarednorman) [reply dev_tools/lib/solidus/release_drafter.rb:71]: Ha, good catch.

**Review by waiting-for-dev — CHANGES_REQUESTED** [review]
Thanks for this clean-up ❤️ 

Let's revert the changes in the `dev_tools` directory out for now, as all blank lines were intentionally there.

**Review by jarednorman — COMMENTED** [review]
We should also make sure that this commit gets added to the commits that are ignored in blames.

**Comment by waiting-for-dev** [comment]
> We should also make sure that this commit gets added to the commits that are ignored in blames.

See https://github.com/solidusio/solidus/pull/4741 for context

**Pull request closed without merging**